react/src/Participants/components/ParticipantInfoDialog.tsx

```diff
@@ -96,20 +125,31 @@ export default function ParticipantInfoDialog(props: DialogProp) {
                         </div>
                     </>
                 )}
-                {analyses.length > 0 && (
-                    <>
-                        <Divider />
-                        <div className={classes.infoSection}>
-                            <InfoList
-                                infoList={getAnalysisInfoList(analyses)}
-                                title="Analyses"
-                                enums={enums}
-                                icon={<ShowChart />}
-                                linkPath="/analysis"
-                            />
-                        </div>
-                    </>
-                )}
+
+                <>
+                    <Divider />
+                    <Box margin={3}>
+                        <Button
+                            variant="contained"
+                            size="small"
+                            onClick={() => setAnalysisInfoListOpen(!analysisInfoListOpen)}
+                        >
+                            {`${analysisInfoListOpen ? "Hide" : "Show"} Analyses`}
+                        </Button>
+                        <Collapse in={analysisInfoListOpen}>
+                            {analyses && analyses.length ? (
+                                <InfoList
+                                    infoList={getAnalysisInfoList(analyses)}
+                                    enums={enums}
+                                    icon={<ShowChart />}
+                                    linkPath="/analysis"
+                                />
+                            ) : (
+                                <Typography>There are no analyses for this participant</Typography>
```

Comment:
I think this text should display instead of the button if there are no analyses associated to this participant. Alternatively, disable the button if there are no analyses.